### PR TITLE
Make push methods static

### PR DESCRIPTION
**Pull request by danielpassos**
@wei-lee Just found this bug. 

Should we release 2.3.1 or add it on 2.3.0?

### CODE REVIEW

**Comment by danielpassos** [comment]
@wei-lee Just found this bug. 

Should we release 2.3.1 or add it on 2.3.0?

**Comment by wei-lee** [comment]
@danielpassos I think it's better to release it as 2.3.1

**Comment by danielpassos** [comment]
@wei-lee @matzew @secondsun Could you give it a spin?

@wei-lee Should we check if `init` method was called before? Should we check if context is `null`?

**Comment by matzew** [comment]
@danielpassos do we need to update any docs for this too ?

File: fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java [hunk 1]
```diff
@@ -458,11 +457,10 @@ public void pushRegister(Context pContext, FHActCallback pCallback) {
      *
      * This method need to be called <b>after</b> a success {@link #init(android.content.Context, FHActCallback)}
      *
-     * @param pContext  your application's context
      * @param pPushConfig extra configuration for push
      * @param pCallback the pCallback function to be executed after the device registration is finished
      */
-    public void pushRegister(final Context pContext, final PushConfig pPushConfig, final FHActCallback pCallback) {
+    public static void pushRegister(final PushConfig pPushConfig, final FHActCallback pCallback) {
```
Comment (matzew) [inline fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java]: I think the arguments e.g. `pCallback` are poorly named

Comment (matzew) [inline fh-android-sdk/src/main/java/com/feedhenry/sdk/FH.java]: Looks like legacy :-) Nevermind, than

**Comment by matzew** [comment]
+1 on the change, and also +1 on a 2.3.1 release of this.

If needed, let's update the doc for the Android SDK, regarding push

**Comment by secondsun** [comment]
LGTM +1 +1 on 2.3.1

**Comment by danielpassos** [comment]
Thanks @wei-lee, @secondsun, @matzew will merge now and start the `2.3.1` release train tomorrow, if no complaints

**Comment by matzew** [comment]
sounds good to me

**Pull request merged**